src/Compilers/Core/Portable/StrongName/PortableStrongNameProvider.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Cci;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class PortableStrongNameProvider : StrongNameProvider
+    {
+        internal readonly ImmutableArray<string> _keySearchPaths;
+        internal IOOperations IOOp { get; private set; }
```

Comment:
This name is very easy to confuse with IOperations. I'd prefer the name IFileSystem

Reply:
I've renamed it to `IStrongNameFileSystem`.

Should it still have `I` in front even though it's just a class?